addons/binding/org.openhab.binding.loxone/META-INF/MANIFEST.MF

```diff
@@ -1,47 +1,47 @@
-Manifest-Version: 1.0
-Bundle-ActivationPolicy: lazy
-Bundle-ClassPath: .
-Bundle-ManifestVersion: 2
-Bundle-Name: Loxone Binding
-Bundle-RequiredExecutionEnvironment: JavaSE-1.8
-Bundle-SymbolicName: org.openhab.binding.loxone;singleton:=true
-Bundle-Vendor: openHAB
-Bundle-Version: 2.2.0.qualifier
-Export-Package: 
- org.openhab.binding.loxone,
- org.openhab.binding.loxone.handler
-Import-Package: 
- com.google.common.collect,
- com.google.gson,
- com.google.gson.annotations,
- org.apache.commons.codec,
- org.apache.commons.codec.binary,
- org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.jetty.http,
- org.eclipse.jetty.util,
- org.eclipse.jetty.util.component,
- org.eclipse.jetty.websocket.api,
- org.eclipse.jetty.websocket.api.annotations,
- org.eclipse.jetty.websocket.api.extensions,
- org.eclipse.jetty.websocket.client,
- org.eclipse.jetty.websocket.common,
- org.eclipse.jetty.websocket.common.extensions,
- org.eclipse.smarthome.config.core,
- org.eclipse.smarthome.config.discovery,
- org.eclipse.smarthome.core.common,
- org.eclipse.smarthome.core.library.types,
- org.eclipse.smarthome.core.thing,
- org.eclipse.smarthome.core.thing.binding,
- org.eclipse.smarthome.core.thing.binding.builder,
- org.eclipse.smarthome.core.thing.link,
- org.eclipse.smarthome.core.thing.type,
- org.eclipse.smarthome.core.types,
- org.eclipse.smarthome.io.transport.upnp,
- org.jupnp,
- org.jupnp.model.meta,
- org.jupnp.model.types,
- org.openhab.binding.loxone,
- org.openhab.binding.loxone.handler,
- org.osgi.framework,
- org.slf4j
-Service-Component: OSGI-INF/*.xml
+Manifest-Version: 1.0
```

Comment:
This is not a proper resolving of this merge conflict, please follow the new sorting / indentation etc and only add your new imports

Reply:
Did you address this one?

Reply:
Yes you did, only github did not show that properly :-)